setup.cfg

```diff
@@ -1,3 +1,15 @@
+[metadata]
+name = ilastik-core
+description = ilastik - the interactive learning and segmentation toolkit
+url = https://github.com/ilastik/ilastik
+
+[options]
+packages = find:
+include_package_data = True
+
+[options.packages.find] #optional
+include=ilastik*, lazyflow*
+
 [tool:pytest]
```

Comment:
Move  pytest and isort configs to `pyproject.toml`?

Reply:
note, pytest config still needs to stay in setup.cfg (we're still stuck with <4 - pytproject is supported in 6+)